doc/dev/ref-manual.md

```diff
@@ -119,16 +114,16 @@ In CHERIoT Safe the size of the revocation tag memory is 16 KiB.
 ### List of SRAM blocks
 
 Here's a list of blocks by size that we need to allocate in SRAM.
-The XC7A35T has 100 blocks of 18 kilobit block RAM, [see datasheet](https://docs.xilinx.com/v/u/en-US/ds180_7Series_Overview).
-In total that gives use 225 KiB of block RAM, but we may not efficiently map onto 18 kilobit blocks and thus lose some memory.
-The block RAM usage in the table below was calculated using Vivado 2023.2's block memory generator.
+The XC7A50T has 150 blocks of 18 kilobit block RAM, [see datasheet](https://docs.xilinx.com/v/u/en-US/ds180_7Series_Overview).
+In total that gives use 337 KiB of block RAM, but we may not efficiently map onto 18 kilobit blocks and thus lose some memory.
+The block RAM usage in the table below was calculated using Vivado 2024.1's block memory generator.:
```

Comment:
You have to pick a side Marno: `.` or `:`.

Reply:
Haha, yes!